Subject: simple_none.c needs '#include "config.h"'
From: Michael Olbrich <m.olbrich@pengutronix.de>

simple_none.c uses HAVE_SOFT_FLOAT it has to include config.h
for this to work properly.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/mixer/simple_none.c |    1 +
 1 file changed, 1 insertion(+)

Index: b/src/mixer/simple_none.c
===================================================================
--- a/src/mixer/simple_none.c
+++ b/src/mixer/simple_none.c
@@ -40,6 +40,7 @@
 #include <limits.h>
 #include <alsa/asoundlib.h>
 #include "mixer_simple.h"
+#include "config.h"
 
 #ifndef DOC_HIDDEN
 
